FirebaseAuth/Sources/Swift/AuthProvider/PhoneAuthProvider.swift

```diff
@@ -19,7 +19,7 @@ import Foundation
 ///
 /// This class is available on iOS only.
 @available(iOS 13, tvOS 13, macOS 10.15, macCatalyst 13, watchOS 7, *)
-@objc(FIRPhoneAuthProvider) open class PhoneAuthProvider: NSObject {
+@objc(FIRPhoneAuthProvider) open class PhoneAuthProvider: NSObject, @unchecked Sendable {
```

Comment:
Add a todo here if you plan on eventually making it not unchecked.

Reply:
Thanks, will do. I'm thinking that for Firebase 12, we mark all `open` classes as `public final` (unless external subclassing is expected).